napari/_qt/widgets/qt_dask_settings.py

```diff
@@ -0,0 +1,315 @@
+from qtpy.QtCore import Signal
+from qtpy.QtWidgets import QCheckBox, QHBoxLayout, QLabel, QSpinBox, QWidget
+
+from ...utils.translations import trans
+
+
+class QtDaskSettingsWidget(QWidget):
```

Comment:
This class seems like **a lot** of code for what it's doing... My feeling is that we could/should implement this as a magicgui widget with a float value. That might not be feasible within this one PR, but basically I'd like to have a clear road towards simpler, even declarative classes for preferences. If every bool + float preference engenders this much code, preferences become a maintenance nightmare that I'd prefer (heh 😜) not to deal with...

Reply:
This point was addressed by @ppwadhwa doing a refactor to make: 

- https://github.com/napari/napari/blob/dbd7b9d32ebd44b15d83dcde0609ae2a8a4c5d4a/napari/_qt/widgets/magic_gui_base_widget.py
- https://github.com/napari/napari/blob/dbd7b9d32ebd44b15d83dcde0609ae2a8a4c5d4a/napari/_qt/widgets/magic_gui_dask_settings.py